### PR TITLE
DEV-3415-new-go-exercise-slice-remove

### PR DESCRIPTION
**Pull request by Hamzaelkhatri**

### CODE REVIEW

File: tests/sliceremove_test/main.go [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+)
+
+func main() {
+	args := [][][]int{
+		{{1, 2, 3, 4, 5}, {1}},
+		{{1, 2, 3, 4, 5}, {5}},
+		{{1, 2, 3, 4, 5}, {2}},
+		{{}, {1}},
+		{{1}, {1}},
+		{{1, 3, 4}, {2}},
```
Comment (tiagocollot) [inline tests/sliceremove_test/main.go:17]: This edge case doesn't make sense. We can remove it.

File: tests/sliceremove_test/main.go [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+	"github.com/01-edu/go-tests/lib/challenge"
+	"github.com/01-edu/go-tests/solutions"
+)
+
+func main() {
+	args := [][][]int{
+		{{1, 2, 3, 4, 5}, {1}},
+		{{1, 2, 3, 4, 5}, {5}},
+		{{1, 2, 3, 4, 5}, {2}},
+		{{}, {1}},
```
Comment (tiagocollot) [inline tests/sliceremove_test/main.go:15]: This edge case doesn't make sense. We can remove it.

**Review by tiagocollot — CHANGES_REQUESTED** [review]

File: tests/sliceremove_test/main.go (hunk 2 above)
Comment (jotapero) [reply tests/sliceremove_test/main.go:15]: It covers in case a wrong input is given. I think it can stay

File: tests/sliceremove_test/main.go (hunk 1 above)
Comment (jotapero) [reply tests/sliceremove_test/main.go:17]: I think it covers an answer that doesn't happen and can stay

File: tests/sliceremove_test/main.go (hunk 2 above)
Comment (jotapero) [reply tests/sliceremove_test/main.go:15]: same as above

**Review by tiagocollot — APPROVED** [review]
LGTM! ⚡
